x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/add_source_logic.ts

```diff
@@ -407,9 +398,7 @@ export const AddSourceLogic = kea<MakeLogicType<AddSourceValues, AddSourceAction
         ? `/internal/workplace_search/org/sources/${serviceType}/prepare`
         : `/internal/workplace_search/account/sources/${serviceType}/prepare`;
 
-      const query = {
-        kibana_host: kibanaHost,
-      } as HttpFetchQuery;
+      const query = {} as HttpFetchQuery;
```

Comment:
As with below, we can omit the query and on line 407 below, you can just perform the `GET`:

```JavaScript
const response = await HttpLogic.values.http.get<SourceConnectData>(route);
```